### ISSUE TITLE
AttributeError: 'Blockchain' object has no attribute 'get_checkpoint'

### ISSUE THREAD

**Issue by ProfessorVenkman** [fyookball/electrum#815]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
  File "/Users/neil/src/electron-cash/gui/qt/network_dialog.py", line 61, in on_update
    self.nlayout.update()
  File "/Users/neil/src/electron-cash/gui/qt/network_dialog.py", line 362, in update
    checkpoint = chain.get_checkpoint()

AttributeError: 'Blockchain' object has no attribute 'get_checkpoint'
```

Reporter
------------

This issue was reported by 1 user(s):

| Electrum Version  | Operating System  | Wallet Type  | Locale |
|---|---|---|---|
| 3.3.1  | Darwin-17.7.0-x86_64-i386-64bit | standard | en_GB |

Additional Information
------------------------

The reporting user(s) did not provide additional information.

**Comment by kyuupichan** [comment]
@rt121212121 I assume this is your changes?

**Comment by kyuupichan** [comment]
Traceback (most recent call last):
  File "/Users/neil/src/electron-cash/lib/util.py", line 153, in run_jobs
    job.run()
  File "/Users/neil/src/electron-cash/lib/verifier.py", line 77, in run
    self.undo_verifications()
  File "/Users/neil/src/electron-cash/lib/verifier.py", line 149, in undo_verifications
    height = self.blockchain.get_checkpoint()
AttributeError: 'Blockchain' object has no attribute 'get_checkpoint'

**Comment by rt121212121** [comment]
Yes, will fix in a second, been waiting for end of working day.

**Comment by rt121212121** [comment]
Fixed in 4fb0f90a004d37fd9f204659aa468c602fc754db.
